### PR TITLE
Qr code reading

### PR DESCRIPTION
**Pull request by Gerussi**
### What

The user can now read a QR code by clicking on the "Scan QR Code" in the main 

### To Do

Complete with the implementation of "follow" in the MainNavActivity/barecodeLauncher/createPopUp method. 
For now, the createPopUp method does not allow to displays personnalized strings, so the name of the user cannot be display. Also, it does not allow to display images, so the user image cannot be displayed

### CODE REVIEW

**Review by atappy — APPROVED** [review]
great job ! still some tests missing, but will likely be solved in subsequent PRs.

File: app/src/main/java/com/github/palFinderTeam/palfinder/ProfileActivity.kt [hunk 1]
```diff
@@ -126,6 +136,26 @@ class ProfileActivity : AppCompatActivity() {
...
     }
 
+    /**
+     * Clicking on QR Code icon will show QR code
+     */
+    fun showQR(view: View?) {
+        //Initiate the barcode encoder
+        val barcodeEncoder = BarcodeEncoder()
+        //Encode text in editText into QRCode image into the specified size using barcodeEncoder
+        val bitmap = barcodeEncoder.encodeBitmap(intent.getStringExtra(USER_ID), BarcodeFormat.QR_CODE, 512, 512)
```
Comment (TruddyTheDuddi) [inline app/src/main/java/com/github/palFinderTeam/palfinder/ProfileActivity.kt:146]: This number could be written somewhere maybe in values.xml, string, or as a companion object

**Review by TruddyTheDuddi — APPROVED** [review]
Nice job!
Though, the test are missing, if they don't take too much time and you still have some hours to work on this PR, it maybe should be looked into!

File: app/src/main/java/com/github/palFinderTeam/palfinder/ProfileActivity.kt (hunk 1 above)
Comment (Gerussi) [reply app/src/main/java/com/github/palFinderTeam/palfinder/ProfileActivity.kt]: Done :)

**Pull request merged**